### PR TITLE
MM-T569 Integrations Page resize and check links

### PR DESCRIPTION
**Pull request by MikeworX**
**_Summary_**

This PR validates that the integration's page sidebar and icon links are visible and continue to function when the browser window is shrunk down to a smaller size (500 x 500) from the default (1300 x 600).

Each element is confirmed to be visible and working, starting with the left side-bar, then moving to the main links.

Screen Shot 2020-10-14 at 11 30 34 AM

**_Links_**

[vercel](https://automation-test-cases.vercel.app/test/MM-T569)
[TM4J](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T569)

### CODE REVIEW

File: e2e/cypress/integration/integrations/integrations_page_spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+describe('Integrations', () => {
+    let teamA;
+
+    before(() => {
+        // # Login, create incoming webhook for Team A
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/integrations_page_spec.js:16]: This seems to be not accurate. How about?
```suggestion
        // # Setup with the new team
```

File: e2e/cypress/integration/integrations/integrations_page_spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        // * Slash Commands link is visible and works
+        cy.get('.integrations-list.d-flex.flex-wrap').within(() => {
+            cy.findByText('Slash Commands').scrollIntoView().should('be.visible').click();
+            cy.url().should('include', '/commands');
+        });
+
+        // # Return to integrations home
+        cy.visit(`/${teamA}/integrations`);
+
+        // * Bot Accounbts link is visible and works
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/integrations_page_spec.js:80]:
```suggestion
        // * Bot Accounts link is visible and works
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @MikeworX. LGTM, except for a couple of suggestions.

**Review by saturninoabril — APPROVED** [review]
Thanks @MikeworX! LGTM.

**Review by prapti — APPROVED** [review]
Looks great!

**Pull request merged**